use-timescale/security/multi-factor-authentication.md

```diff
@@ -30,8 +30,8 @@ Take the following steps to configure two-factor authentication:
 <Procedure>
 
 1.  Log in to [$CONSOLE][cloud-login] with your username and password. 2FA is not available if you log in with Google SSO. 
-1.  Click the `User name` icon in the bottom left of $CONSOLE and select `Account`.
-1.  In `Account`, click `Add two-factor authentication`.
+1.  Click the `User name` icon in the top right of $CONSOLE and select `Account settings`.
```

Comment:
Click your user name at the top right ... 

And for other times this turns up.